### DISCUSSION TITLE
Difficulties importing sympy

### DISCUSSION THREAD

**Discussion by jprenderg** [sympy/sympy#23910] [General]
Hello,

I am trying to import sympy in Spyder 5.2.2 using:
from sympy import *

This worked a couple of days ago, but now I am getting an error on the import:
"name 'init_printing' is not defined"

Does anyone else have this issue or know of a solution?

**Comment by oscarbenjamin** [comment]
Have you by any chance created a file called `sympy.py` in your current directory?
